tests/conftest.py

```diff
@@ -4,9 +4,9 @@
 
 
 def pytest_collection_modifyitems(config, items):
-    """Automatically skip tests marked with s2_api_request when S2_API_KEY is not set."""
-    if not os.getenv("S2_API_KEY"):
-        skip_s2 = pytest.mark.skip(reason="S2_API_KEY is not set")
+    """Automatically skip tests marked with asta_tool_request when ASTA_TOOL_KEY is not set."""
+    if not os.getenv("ASTA_TOOL_KEY"):
+        skip_asta = pytest.mark.skip(reason="ASTA_TOOL_KEY is not set")
         for item in items:
-            if "s2_api_request" in item.keywords:
-                item.add_marker(skip_s2)
+            if "asta_tool_request" in item.keywords:
```

Comment:
nit: my one minor gripe with `asta_tool_request` is that it could be confused with a request to any asta(bench) tool, but really it's only about the asta corpus tools (because they require authentication); consider `asta_tool_api_request` if you agree, but otherwise I don't think it's a crucial change.

Reply:
Changed to `asta_tool_api_request`